pennylane/labs/resource_estimation/resource_operator.py

```diff
@@ -91,10 +92,11 @@ def set_resources(cls, new_func: Callable) -> None:
         """Set a custom resource method."""
         cls.resources = new_func
 
-    @abstractmethod
+    @property
```

Comment:
Just double-checking that this does not cause issues elsewhere.

Reply:
Nice catch Soran, this was fixed 👍🏼